go/host/storage/hostdb/block.go

```diff
@@ -23,7 +24,7 @@ func AddBlock(dbtx *dbTransaction, statements *SQLStatements, b *types.Header) e
 		b.Hash().Bytes(), // hash
 		header,           // l1 block header
 	)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), " UNIQUE constraint failed") {
```

Comment:
I noticed in the tests that it was erroring with the unique constraint.
Race condition. Probably the same block submitted twice

Reply:
Is it safe to ignore that error though? Apart from the tests, this code is called by storage.go AddBlock() and it will try write the dbTx if it doesn't error - not sure if there's any side effect of that.